### ISSUE TITLE
Reminders tab not showing!

### ISSUE THREAD

**Issue by yaz94** [uphy/obsidian-reminder#186]
How can I show the tab/sidebar for my set reminders? The "clock" icon shows the recent files instead of reminders!

Reminder 1.1.15
Ubunut 22.04
Obsidian v1.5.12

**Comment by erottensteiner** [comment]
Yes, command "Reminder: Show reminders" shows no tab with the results!
Reminder 1.1.15
Windows 11
best regards

**Comment by jijivski** [comment]
can not receive remind and can not set the tab,
Reminder 1.1.15
Windows 10
obsidian 1.6.3
All the best, just found this extension, I think this would be wonderful to me.

**Comment by Vasdranna** [comment]
Same here, both `show` and `scan` commands doesn't seem to work at all...

**Comment by shishi** [comment]
same, do nothing "Show reminders"

Obsidian 1.8.10
Windows 10
Reminder 1.1.19
